lib/graphql/persisted_queries/store_adapters/redis_store_adapter.rb

```diff
@@ -7,22 +7,27 @@ module PersistedQueries
     module StoreAdapters
       # Redis adapter for storing persisted queries
       class RedisStoreAdapter < BaseStoreAdapter
-        def initialize(redis_client:)
+        DEFAULT_EXPIRATION = 24 * 60 * 60
+        DEFAULT_NAMESPACE = "graphql-persisted-query"
+
+        def initialize(redis_client:, expiration: DEFAULT_EXPIRATION, namespace: DEFAULT_NAMESPACE)
           @redis_proc = build_redis_proc(redis_client)
+          @expiration = expiration
```

Comment:
We probably should set the default value for `expiration` parameter to `nil` and change this line to `@expiration = expiration || DEFAULT_EXPIRATION`, which should help us avoid accidental nils (e.g., `use GraphQL::PersistedQueries, expiration: nil` or a bit more cryptic `use GraphQL::PersistedQueries, expiration: ENV["REDIS_EXPIRATION"]`)

Reply:
Good catch.  I had considered it, but didn't give it enough weight.  I'll make this change now.